docs/guides/installing-unreleased-changes.md

```diff
@@ -6,8 +6,8 @@ How to install local Carbon into your application
 
 1. Pull down the Carbon repository locally.
 2. Checkout the appropriate branch which contains the changes you want to install.
-3. Run `npm run-script prepublishOnly` - This will compile the code from ES6/ES7 into ES5 for the browser to process.
-4. Within your local application that uses Carbon. Change the package.json file so that carbon points to a relative path `./../carbon`.
-5. Run `npm install carbon-react` or `npm install` to install the local version of the application.
+3. Run `npm run precompile` - This will compile the code from ES6/ES7 into ES5 for the browser to process.
+4. Run `npm install $(npm pack ./../../carbon/ | tail -1)` to install the local version of the application.
+*Relative path to carbon may be different depending on where it's being installed from.
 
 Note: Optionally you can just run `npm install ./../carbon` within your application to install the changes locally however running `npm install` or `npm install carbon-react` afterwards will override these changes.
```

Comment:
Just noticed this note at the bottom, that can probably be removed now too!

Reply:
Can also remove the `NB` towards the top of the file too

Reply:
Sorry about this, I wasn't 100% sure what needs to stay and what doesn't